github/resource_github_organization_project.go

```diff
@@ -0,0 +1,116 @@
+package github
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	"github.com/google/go-github/github"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceGithubOrganizationProject() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceGithubOrganizationProjectCreate,
+		Read:   resourceGithubOrganizationProjectRead,
+		Update: resourceGithubOrganizationProjectUpdate,
+		Delete: resourceGithubOrganizationProjectDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"body": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"url": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+		},
+	}
+}
+
+func resourceGithubOrganizationProjectCreate(d *schema.ResourceData, meta interface{}) error {
+	client := meta.(*Organization).client
+	o := meta.(*Organization).name
+	n := d.Get("name").(string)
+	b := d.Get("body").(string)
+
+	options := github.ProjectOptions{
+		Name: n,
+		Body: b,
+	}
```

Comment:
Nitpick (non-blocking): I think we could just inline the fields above to the struct to reduce LOC and improve readability a bit.

Reply:
Fixed in f1a04f4, thank you.